modules/ROOT/pages/_partials/commons/common-indexing.adoc

```diff
@@ -109,6 +109,183 @@ include::{root-partials}block_tabbed_code_example.adoc[]
 
 endif::is-c[]
 
+== Array Indexing 
 
+Couchbase Lite 3.2.1 introduces functionality to optimize querying arrays.
+Array `UNNEST` to unpack arrays within a document to allow joins with the parent object, and array indexes for indexing nested array properties and allow more efficient queries with `UNNEST`. 
 
+=== Array UNNEST
 
+You can use `UNNEST` in queries to unpack arrays within a document into individual rows.
+This functionality makes it possible to join them with its parent object in the query.
+
+`UNNEST` is used within the `FROM` clause and can be chained to perform multi-level `UNNEST`.
+
+NOTE: Couchbase currently supports inner `UNNEST` only. 
+
+
+=== Array UNNEST Syntax
+
+The syntax for `UNNEST` is shown below:
```

Comment:
The syntax should be described the same way as the other expressions for consistency. For example:

https://preview.docs-test.couchbase.com/CBL321/couchbase-lite/current/swift/query-n1ql-mobile.html#syntax-18

Reply:
Done!